docs/Project.toml

```
@@ -15,8 +15,8 @@ CategoricalArrays = "0.8, 0.9, 0.10"
 DataFrames = "0.22, 1"
 Documenter = "0.26, 0.27"
 Gadfly = "1.3.4"
-MCMCChains = "5"
+MCMCChains = "6"
 MLJBase = "0.19, 0.20, 0.21"
-MLJDecisionTreeInterface = "=0.3.0"
+MLJDecisionTreeInterface = "0.1.3, 0.2, 0.3"
```

I think 0.3 is sufficient (and the other versions are untested anyway):
```suggestion
MLJDecisionTreeInterface = "0.3"
```